src/_gettsim/policy_environment.py

```diff
@@ -282,6 +284,40 @@ def _parse_einführungsfaktor_vorsorgeaufw_alter_ab_2005(date, params):
     return params
 
 
+def _parse_vorsorg_rv_anteil(date, params):
```

Comment:
Please open an issue that this function will need adjustment for people who are receiving pension benefits while working (at least I'd think that'll be the case).